3MF Core Specification.md

```diff
@@ -437,8 +437,7 @@ Element **\<build>**
 
 ![element build](images/element_build.png)
 
-The \<build> element contains one or more items to manufacture as part of processing the job. A consumer MUST NOT output any 3D objects not referenced by an \<item> element.
-
+The \<build> element could either be empty, in case of a 3MF Document providing only design objects, or contain one or more items to produce as part of processing the job. A consumer MUST manufacture only 3D objects which are referenced by an \<item> element. Any unreferenced object can still be processed by an editing application, which could finalize their manufacturing by referencing them in corresponding \<item> elements.

```

Comment:
I would make this:
"The \<build> element contains zero or more \<item> elements to manufacture as part of processing the job. A consumer MUST NOT manufacture any 3D objects not referenced by an \<item> element. Any unreferenced object can still be consumed by an editing application, which could then reference them in corresponding \<item> elements when they are ready for manufacture."

I think that wording makes it clear that we are supporting new cases without changing any existing workflows.